packages/react-form-renderer/src/use-field-api/use-field-api.js

```diff
@@ -10,7 +10,7 @@ import composeValidators from './compose-validators';
 import isEqual from 'lodash/isEqual';
 
 const calculateInitialValue = (props) => {
-  if (Object.prototype.hasOwnProperty.call(props, 'initialValue') && Object.prototype.hasOwnProperty.call(props, 'dataType')) {
+  if (Object.prototype.hasOwnProperty.call(props, 'initialValue') && props.dataType) {
```

Comment:
Is there any reason why not to keep the `hasOwnProperty` check?

Reply:
Yeah, dataType has to be a string, so we don't have to check falsy values.